### ISSUE TITLE
Restore Window Title Too, Please?

### ISSUE THREAD

**Issue by andy-twosticks** [tmux-plugins/tmux-resurrect#200]
I prefer to work with `setw -g automatic-rename off`.  I notice that the name of the each window is not restored, which is a bit of a pain (pane?).

This is definitely not a bug.  But if Resurrect restored window titles, then it would make me (and presumably a _few_ other people) really happy.

**Comment by bruno-** [comment]
No, this is a bug.. but it was never a problem for me personally and I never got the time to look into it.

**Comment by malkomalko** [comment]
Please see https://github.com/tmux-plugins/tmux-resurrect/pull/211 which fixes this issue when restoring from a base tmux session.
